e2e/test/helpers/RetryOperationHelper.cs

```diff
@@ -20,12 +20,53 @@ public class RetryOperationHelper
         /// This is for E2E tests of provisioning service clients.
         /// </remarks>
         /// <param name="asyncOperation">The async operation to be retried.</param>
-        /// <param name="retryPolicy">The retry policy to be applied.</param>
+        /// <param name="retryPolicy">The retry policy of hub device/module to be applied.</param>
         /// <param name="cancellationToken">The cancellation token to cancel the operation.</param>
         /// 
         public static async Task RunWithRetryAsync(
```

Comment:
I'd recommend giving these unique names so it is more obvious from the caller that we're doing HubService or HubClient retry.

We've had test bugs where we were unknowingly using the wrong retry policy so tests weren't atually retrying.